### DISCUSSION TITLE
How to access revisions table with related collection by id in desc order

### DISCUSSION THREAD

**Discussion by sandyksit** [directus/directus#9845] [Archive]
Hi team,

I am not able to fetch revisions table with related collection by id in desc order, Any idea how to get? I am able fetch only 100 record by end point '/revisions'.  @rijkvanzanten @joselcvarela please help
I am using V9.0.1 version.

**Comment by rijkvanzanten** [comment]
You should be able to use

```
GET /revisions
  ?filter[collection][_eq]=<collection>
  &filter[item][_eq]=<id>
  &sort=-id
```
